mycroft/configuration/mycroft.conf

```diff
@@ -146,6 +146,15 @@
         "ssl": false
   },
 
+  // URIs to use for testing network connection.
+  "network_tests": {
+    "dns_primary": "8.8.8.8",
+    "dns_secondary": "8.8.4.4",
+    "https_url": "https://www.google.com",
```

Comment:
I was just thinking that this isn't explicitly required to use the https protocol so maybe this should be renamed...

Reply:
Yeah, but tricky to come up with a good descriptive name. Maybe `web_url` and a comment?